packages/schematics/angular/e2e/e2e-long.md

```diff
@@ -0,0 +1 @@
+The e2e tests are created in a separate app, next to the project being tested in the `projects` folder of the workspace.
```

Comment:
Maybe: The e2e tests are created in a separate app in the `projects` folder of the workspace, next to the project being tested.